scripts/utils/trading_strategy_helpers.js

```diff
@@ -282,35 +290,6 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return buildBundledTransaction(transactions)
   }
 
-  // TODO - pass stableTokenDecmials and targetTokenDecimals since these are the only attributes used here.
-  const calculateBuyAndSellAmountsFromPrice = function(price, stableToken, targetToken) {
```

Comment:
Careful here, this function is used by other scripts that haven't been touched here. I have expanded on this below.